sirepo/job_supervisor.py

```diff
@@ -168,16 +168,16 @@ def init_module(**imports):
                 "maximum run-time for sequential job",
             ),
         ),
-        purge_non_premium_after_secs=(
-            0,
-            pkconfig.parse_seconds,
-            "expiration period for purging non-premium users simulations",
-        ),
-        purge_non_premium_task_secs=(
+        purge_check_interval=(
             None,
             pkconfig.parse_seconds,
             "time interval to clean up simulation runs of non-premium users (%H:%M:%S)",
         ),
+        run_dir_lifetime=(
+            0,
```

Comment:
I think 0 is a poor choice, because this expires immediately. This could yield unexpected results. I think the minimum should be twice the simulation run period. We don't need to assert this, but maybe put it at 1d here.

Reply:
796209d1cbcd5b1c7da7d74982d9aeeb0a88e993